README.md

```diff
@@ -2,29 +2,34 @@
 
 ## How is FLINT different from earlier integrating tools?
 
-The FLINT is using the lessons learned from first generation tools, to build a new framework that meets present and future needs. The key improvements compared to the first generation tools include:
-+ a flexible, modular approach, allowing for country specific implementations from [IPCC Tier 1 to 3 ](https://www.reddcompass.org/mgd-content-v1/dita-webhelp/en/Box1.html)
-+ support for [wall-to-wall, sample based, jurisdictional, and supply chain approaches](https://static1.squarespace.com/static/5896200f414fb57d26f3d600/t/59362b028419c2db8f57e747/1496722191543/REDD_nested_projects.pdf)
-+ the ability to cover all [land uses and land use changes, and activity-based reporting such as REDD+](https://theredddesk.org/what-redd)
-+ scenario analysis systems to allow the development of projections
-+ the potential to be used for multiple other purposes, including economics, water and biodiversity
-+ development managed using a true open source approach under [moja global](http://moja.global), which will allow users (countries, companies and organizations) to direct strategy and control the budget.
-+ software that allows data processing on local desktops or cloud-based systems   
+FLINT is using the lessons learned from the first generation of tools to build a new framework that meets current and future needs. Compared with the first-generation tools, the main improvements include:
++ A flexible and modular approach that allows specific countries from [IPCC Tier 1 to 3](https://www.reddcompass.org/mgd-content-v1/dita-webhelp/en/Box1.html) to implement a [comprehensive, sample-based, jurisdictional and supply chain-based approach.](https://static1.squarespace.com/static/5896200f414fb57d26f3d600/t/59362b028419c2db8f57e747/1496722191543/REDD_nested_projects.pdf)
++ The ability to cover all [land uses and land use changes, and activity-based reporting such as REDD+.](https://theredddesk.org/what-redd)
++ Scenario analysis systems to allow the development of projections.
++ Potential for a variety of other purposes, including economics, water, and biodiversity.
++ Development managed using a true open source approach under [moja global](http://moja.global), which will allow users (countries, companies and organizations) to direct strategy and control the budget.
++ Software that allows data processing on local desktops or cloud-based systems.  
 
 
-## How to use FLINT?  
+## Getting Started with FLINT
+
+Below are instructions for quick installation.
+
+Note: If you need more detailed information, you can use a [step-by-step guide](https://github.com/moja-global/FLINT/tree/master/How_to_use_FLINT) or view the [video](https://www.youtube.com/watch?v=Jfi2-vEhfkg).
```

Comment:
Put `Note` in bold

Reply:
We can also use this format as well `**_NOTE:_**`  as 

**_NOTE:_** If you need more detailed information ....